.github/actions/wheel/action.yml

```diff
@@ -15,7 +15,10 @@ runs:
       run: |
         echo "BAZEL_CONFIG=manylinux" >> $GITHUB_ENV
         echo "PLAT_NAME=manylinux1_x86_64" >> $GITHUB_ENV
-        echo "REMOTE_CACHE=--remote_cache=http://localhost:8080" >> $GITHUB_ENV
+        if ping -c 1 http://localhost:8080 &> /dev/null
```

Comment:
Cache breaks pretty often, we need to just check if its working at the start and decide to actually use it or not based on that.